.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,29 @@
+name: Pull Request
+about: Submit a pull request to improve Harbor Satellite
+title: ''
+labels: ''
+assignees: ''
+---
+
+## Description
+
+Briefly describe the purpose of this PR and what it changes. If it addresses an issue, please reference it.
+
+## Checklist
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)